src/test/ReactTestUtils.js

```diff
@@ -269,9 +269,8 @@ var ReactTestUtils = {
       componentType
     );
     if (all.length !== 1) {
-      throw new Error(
-        'Did not find exactly one match for componentType:' + componentType +
-        ' (found ' + all.length + ')'
+      throw new Error('Did not find exactly one match ' +
```

Comment:
Can you leave the newline after the paren as it was before to match the surrounding code? Also please squash your commits together.